themes/developer/developer.theme.sh

```diff
@@ -0,0 +1,229 @@
+#! bash oh-my-bash.module
+
+SCM_THEME_PROMPT_DIRTY=" ${_omb_prompt_brown}✗"
+SCM_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" ${_omb_prompt_green}|"
+SCM_THEME_PROMPT_SUFFIX="${_omb_prompt_green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${_omb_prompt_brown}✗"
+GIT_THEME_PROMPT_CLEAN=" ${_omb_prompt_bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${_omb_prompt_green}|"
+GIT_THEME_PROMPT_SUFFIX="${_omb_prompt_green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+function return_delimited {
```

Comment:
Can you prefix `_omb_theme_developer_` to all the function names specific to this theme?

Reply:
cd7e914